src/components/app-header/app-header.tsx

```diff
@@ -199,16 +199,19 @@ export const AppHeader = ({
           />
         )}
         <IconContainer>
-          <Button
-            flavour="borderless"
-            neutral
-            themeType="dark"
-            className="btn"
-            data-toggle="modal"
-            data-target="#helpModal"
-            icon="question"
-            title="Need help?"
-          />
+          <Documentation app="agent">
+            {(toggle: any) => (
+              <Button
+                flavour="borderless"
+                neutral
+                themeType="dark"
+                className="btn"
+                icon="question"
+                onClick={toggle}
```

Comment:
what do you think about using `onToggle` or `handleToggle` naming convention?
Especially since we'll remove typescript, IMO it would be good to have some hint what is type is that
cc @allelos @novykh 
https://medium.com/javascript-in-plain-english/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364

Reply:
I think that it depends on the case. For instance, the `toggle` verb cannot be used as a standalone variable like `loading, loaded` etc., it describes a reaction, so the `onToggle` seems redundant to me. 

We can take this example
https://github.com/streamich/react-use/blob/master/docs/useToggle.md
```
const [on, toggle] = useToggle(true);
..onClick={toggle}..
```

Since it is not something major, we could also wait for the other's opinion.